tests/unittest_prioq.c

```diff
@@ -1,10 +1,24 @@
 #include <check.h>
 
 #include <stdlib.h>
+#include <stdio.h>
 
 #include "prioq.h"
+#include "gen_allocdefs.h"
 
-START_TEST(test_prioq_empty)
+/* TODO
+- write tests for gen_alloc_readyplus
+- what's _internal?
+- what's ta stand for?
+- what's a good site for c language reference?
+- what's a good book to read more about c language?
+- what's a good code standard convention to follow?
+- what's a good way to run a single test case of single unit tests
```

Comment:
- what's a good site for c language reference?
cppreference.com is covering plenty of C (and C++), but a good reference remains the C standard itself:
https://stackoverflow.com/questions/17014835/where-can-one-find-the-c89-c90-standards-in-pdf-format

- what's a good code standard convention to follow?
For a new project, I would advise either [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html) or [KNF](http://man.openbsd.org/style.9) (Kernel Normal Form) used by BSDs.
For an existing project, in particular if collaborating, I would preserve the original coding style of that project as much as possible. Which is a bit uncommon for qmail.

- what's a good way to run a single test case of single unit tests
If a test case takes too much time or produces too much output and you want to avoid it, you could split it away in another test file for instance.